app/assets/stylesheets/ppd.scss

```diff
@@ -167,7 +168,7 @@
   }
 
   .transaction-category {
-    background-color: lighten( $lr-logo-grey, 20% );
+    background-color: lighten( #a7a5a6, 20% );
```

Comment:
What was the issue here?

Reply:
The variable value was lost on import/compilation and this was felt to be the quickest way to resolve. Additional ticket on this was requested.

Reply:
Does the other ticket exist @bogdanadrianmarc @jonrandahl ?

Reply:
@joescottdave Had a discussion with @jonrandahl the other day about this, I've put a summary of it in here now: https://github.com/epimorphics/lr_common_styles/issues/75

Reply:
Great 👍